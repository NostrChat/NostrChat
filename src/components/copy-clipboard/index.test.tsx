import {render} from '@testing-library/react';

import CopyToClipboard from './index';

test('1 Default render', () => {
    const view = render(<CopyToClipboard copy="xxx11"><span>click here</span></CopyToClipboard>);
    expect(view.container).toMatchSnapshot();
});
