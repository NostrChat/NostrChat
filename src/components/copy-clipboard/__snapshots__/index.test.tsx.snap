// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1 Default render 1`] = `
<div>
  <span
    aria-label="Copied"
    class=""
    data-mui-internal-clone-element="true"
    style="cursor: pointer;"
  >
    click here
  </span>
</div>
`;
